9jaswag/PostIt
# Implement Feedback

#### What does this PR do?
-Add more API test to increase coverage
- Fix app redirect/reloading bug
- Add client tests

#### Description of Task to be completed?
- Fix component routing
- Cover more API tests

#### How should this be manually tested?
- Check coverall badge for new coverage
- Visit app to see it doesn't reload anymore

#### What are the relevant pivotal tracker stories?
- none
